Graphics/Vulkan/VulkanGraphicsCmdList.cpp

```diff
@@ -0,0 +1,244 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root
+// for more information.
+
+#include <NovelRT/Graphics/Vulkan/VulkanGraphicsBuffer.hpp>
+#include <NovelRT/Graphics/Vulkan/VulkanGraphicsCmdList.hpp>
+#include <NovelRT/Graphics/Vulkan/VulkanGraphicsContext.hpp>
+#include <NovelRT/Graphics/Vulkan/VulkanGraphicsDescriptorSet.hpp>
+#include <NovelRT/Graphics/Vulkan/VulkanGraphicsPipeline.hpp>
+#include <NovelRT/Graphics/Vulkan/VulkanGraphicsRenderPass.hpp>
+#include <NovelRT/Graphics/Vulkan/VulkanGraphicsTexture.hpp>
+
+namespace NovelRT::Graphics::Vulkan
+{
+    VulkanGraphicsCmdList::VulkanGraphicsCmdList(std::shared_ptr<VulkanGraphicsContext> context, VkCommandBuffer commandBuffer) noexcept
+        : _context(context), _commandBuffer(commandBuffer)
+    {
+    }
+
+    std::shared_ptr<VulkanGraphicsContext> VulkanGraphicsCmdList::GetContext() const noexcept
+    {
+        return _context;
+    }
+
+    void VulkanGraphicsCmdList::CmdBeginRenderPass(std::shared_ptr<VulkanGraphicsRenderPass> targetPass,
+                                                   Utilities::Misc::Span<const ClearValue> clearValues)
+    {
```

Comment:
Um... is this cursed formatting? Or is it just GH being funny?
Something has sent clearValues to the right side of the screen LMAO
![image](https://github.com/user-attachments/assets/e5518eb8-eb44-4e61-aa30-c64a93e838fc)

Reply:
it's not doing that on my monitor?